src/util/taproot.rs

```diff
@@ -683,13 +683,12 @@ impl TaprootMerkleBranch {
             Err(TaprootError::InvalidMerkleTreeDepth(sl.len() / TAPROOT_CONTROL_NODE_SIZE))
         } else {
             let inner = sl
-                // TODO: Use chunks_exact after MSRV changes to 1.31
-                .chunks(TAPROOT_CONTROL_NODE_SIZE)
+                .chunks_exact(TAPROOT_CONTROL_NODE_SIZE)
                 .map(|chunk| {
                     sha256::Hash::from_slice(chunk)
-                        .expect("chunk exact always returns the correct size")
-                })
-                .collect();
+                        .expect("chunks_exact always returns the correct size")
+                }).collect();
```

Comment:
Nit: since all other methods start at separate lines it feels natural for `.collect();` to be on a separate line as well. IDK what rustfmt says about it, I just use that style. :)

Reply:
oh yes, you are correct. Thanks, will fix and force push.